test/unit/plugins/doca_memos/README.md

```
@@ -0,0 +1,278 @@
+    // Create engine
+    createEngine();
+
+    // Execute test logic
+    // ...
+
+    // Verify expectations
+    EXPECT_EQ(actual, expected);
+}
+```
+
+2. **Configure mocks** as needed:
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)